.github/workflows/.build.yml

```diff
@@ -10,12 +10,9 @@ on:
       envs:
         required: false
         type: string
-      push:
-        required: false
-        type: boolean
       release:
         required: false
-        type: boolean
+        type: string
```

Comment:
Shame that Github doesn't support `choice` in reusable workflows 🙈 

Should we have an additional `verify` job that would assert that this is has expected values?

Reply:
We can still check if its any of `"" | "pushonly" |  "draft" | "prerelease" |  "release"`